resolver/resolver.go

```diff
@@ -2294,6 +2579,29 @@ func (r *Resolver) resolveMessageCELValues(ctx *context, env *cel.Env, msg *Mess
 			r.resolveMessageValidationErrorDetailCELValues(ctx, env, msg, valIdx, detIdx, detail)
 		}
 	}
+	for idx, varDef := range msg.Rule.VariableDefinitions {
+		if varDef.Expr == nil {
```

Comment:
Is ignoring `varDef.If` intentional? I know the feature is not supposed to be supported in this PR but it seems like the other CELValue fields are resolved here, I just wanted to make sure...